docs/stdlib/datetime.rst

```diff
@@ -652,6 +652,10 @@ EdgeDB stores and outputs timezone-aware values in UTC.
         db> select datetime_current();
         {<datetime>'2018-05-14T20:07:11.755827Z'}
 
+    This function is volatile since it always returns the current time when it
+    is called. As a result, it cannot be used in :ref:`computed properties
```

Comment:
@jaclarke I notice that my links to `ref_datamodel_computed` go to the correct page, but they scroll to the bottom of the page when I'm testing locally. Is that something I'm doing wrong, or is it a bug? The target is at the very top of that "Computed properties" page.